iocage_lib/cache.py

```diff
@@ -42,10 +42,13 @@ def iocage_activated_pool_internal(self, lock=True):
                         all_properties([p for p in pools], types=['filesystem'])
                     )
                 for p in filter(
-                    lambda p: self.dataset_data.get(p, {}).get('org.freebsd.ioc:active') == 'yes',
-                    pools
+                    lambda p: (
+                        p.get('org.freebsd.ioc:active') == 'yes' and p.get('mounted') == 'yes' and
+                        p.get('encryption', 'off') != 'off' and p.get('keystatus', 'available') != 'available'
```

Comment:
Can you please elaborate what pools are we filtering here. Iocage-related, mounted, encrypted, with unavailable keystatus?